ReactWindows/ReactNative.Shared/Bridge/ReactBridge.cs

```diff
@@ -56,6 +56,7 @@ public class ReactBridge : IReactBridge
         public void CallFunction(string moduleName, string method, JArray arguments)
         {
             var response = _jsExecutor.CallFunctionReturnFlushedQueue(moduleName, method, arguments);
+            _reactCallback.IncrementPendingJSCalls();
             ProcessResponse(response, true);
```

Comment:
Why are we incrementing after the _synchronous_ call to JavaScript is completed? Shouldn't this occur before the JavaScript call?

This may just be a problem with the name of the function (see other comment).  When I read this code, it makes me think I should increment the pending JS calls _before_ doing any work in JS.

Reply:
I went back and forth with this before committing, using upstream as a reference. I'm okay to change it.

Reply:
#fixed